sklearn/tree/_classes.py

```diff
@@ -686,9 +686,76 @@ def feature_importances_(self):
             Normalized total reduction of criteria by feature
             (Gini importance).
         """
+        return self._unnormalized_feature_importances / np.sum(
+            self._unnormalized_feature_importances
+        )
```

Comment:
Same remark as above in `BaseForest`. Maybe we can store `all_importances = self._unnormalized_feature_importances` to avoid computing it twice.